Examples/Movies/MovieCell.js

```diff
@@ -24,6 +24,7 @@ var {
   Text,
```

Comment:
And if they do affect UWP, then we'll need to ensure that MovieCell.js is split into a base class and platform specific files.